opwen_email_server/utils/email_parser.py

```diff
@@ -97,6 +142,41 @@ def _get_image_type(response: Response, url: str) -> Optional[str]:
     return content_type
 
 
+def _get_max_sizes() -> Tuple:
+    max_width = config.MAX_WIDTH_IMAGES
+    max_height = config.MAX_HEIGHT_IMAGES
+    return max_width, max_height
+
+
+def _is_size_already_small(size: Tuple) -> bool:
+    width = size[0]
+    height = size[1]
+    max_width = _get_max_sizes()[0]
+    max_height = _get_max_sizes()[1]
+    if width <= max_width and height <= max_height:
```

Comment:
You can just inline this condition here for simplicity: `return width <= max_width and height <= max_height`.